client/my-sites/github-deployments/components/repositories/repository-list.tsx

```diff
@@ -63,6 +63,25 @@ export const GitHubBrowseRepositoriesList = ( {
 		return <NoResults manageInstallationUrl={ installation.management_url } />;
 	}
 
+	const getRepositoryPermissionsNotice = () => {
+		const noticeText =
+			installation.repository_selection === 'all'
+				? __( 'Need to adjust permissions?' )
+				: __( 'Missing GitHub repositories?' );
+
+		const linkText =
+			installation.repository_selection === 'all'
+				? __( 'Update them on GitHub' )
+				: __( 'Adjust permissions on GitHub' );
+
+		return (
+			<p className="github-repositories-list-permissions-notice">
+				{ noticeText }{ ' ' }
+				<ExternalLink href={ installation.management_url }>{ linkText }</ExternalLink>
+			</p>
+		);
```

Comment:
I think maybe we can short circuit the repository selection, it'd be easier to read:

```ts
if(installation.repository_selection === 'all') {
    return <p>Need to adjust permissions? ...</p>
}

return <p>Missing GitHub repositories? ...</p>
```

That looks easier to read to me.